portal-impl/src/portal.properties

```diff
@@ -10276,6 +10276,17 @@
         mail.session.mail.smtp.password,\
         tunneling.servlet.shared.secret
 
+##
+## Analytics Cloud
+##
+
+    #
+    # Analytics Cloud SDK Version
+    #
+    # Env: LIFERAY_ANALYTICSCLOUD_PERIOD_SDK_PERIOD_VERSION
+    #
+    analyticscloud.sdk.version=PROD
```

Comment:
Can we use `analytics.cloud.client.js.version` ? So it's closer to its [module name](https://github.com/liferay-ac/liferay-portal/tree/master/modules/apps/analytics/analytics-client-js)?

Reply:
Sure thing, thanks @marcellustavares